infra/resources/modules/commons/api/ts_management/v1/_openapi.yaml

```diff
@@ -315,6 +376,10 @@ components:
       type: string
       description: Unique identifier of the trial.
       minLength: 1
+    NextTrialId:
+      type: string
+      description: Unique identifier of the next trial.
+      minLength: 1
```

Comment:
**suggestion**: we can call it just `NextId` and reuse the schema for other endpoints in future

Reply:
Renamed to nextId in 20f69ea2658432a88daecb105c2db8f5f1408cac